src/sql/ConfigEditor/ConfigSelect.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { DataSourcePluginOptionsEditorProps, SelectableValue } from '@grafana/data';
+import { AwsAuthDataSourceJsonData, AwsAuthDataSourceSecureJsonData } from '../../types';
+import { ResourceSelector } from '../ResourceSelector';
+import { set, get } from 'lodash';
+
+export interface ConfigSelectProps
+  extends DataSourcePluginOptionsEditorProps<AwsAuthDataSourceJsonData, AwsAuthDataSourceSecureJsonData> {
+  jsonDataPath: string;
+  fetch: () => Promise<Array<string | SelectableValue<string>>>;
+  dependencies?: string[];
+  label?: string;
+  'data-testid'?: string;
```

Comment:
Do you need the test id? Wouldn't using getByLabelTest suffice?

Reply:
It doesn't, it's an issue with grafana/ui (I guess I should create an issue for this?). You get this error using `getByLabelText`:

```
    TestingLibraryElementError: Found a label with the text of: <>, however no form control was found associated to that label. Make sure you're using the "for" attribute or "aria-labelledby" attribute correctly.
```

Also, the test-id is used in the e2e test (the flows only work with aria-labels and test-ids)